### PR TITLE
Write-up for event

### PR DESCRIPTION
**Pull request by cdepillabout**
Here is a write-up for Nix stuff at ZuriHac.

After getting feedback, I plan on posting this to the [NixOS Discourse](https://discourse.nixos.org/) and deleting this repo.

------------------------

edit: I posted this on the NixOS Discourse: https://discourse.nixos.org/t/nix-and-haskell-at-zurihac-2021/13417

### CODE REVIEW

**Comment by cdepillabout** [comment]
Ping @sternenseemann about this as well, although I think he said he was busy this weekend.

**Comment by sternenseemann** [comment]
Not sure how much I'll be able to be around, but I maybe should register…

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+The Nixpkgs Haskell team will be running a Nix-related event at the hackathon.  We will be answering questions about Nix all weekend.  We will be available for voice chats (with screen sharing) to help people figure any Nix- and Haskell-related problems they may be running into.
+
+We hope to help people with the following types of things:
+
+- converting a personal Haskell project to build with Nix
+
+- investigating whether Nix would be a good solution for a project at work
+
+- installing NixOS and setting up a Haskell development environment
```
Comment (maralorn) [inline README.md:15]:
```suggestion
- installing Nix or NixOS

- converting a personal Haskell project to build with Nix

- setting up a Haskell development environment with Nix

- investigating whether Nix would be a good solution for a project at work
```

**Review by maralorn — APPROVED** [review]
I always feel weird making suggestions to wording of native speakers.

I have a few minor content clarification. The rest are some ideas to make the text a bit more enthusiastic while adding some variations to not introduce to much word repititions. Feel free to adopt only what you like!

I wonder how we should announce it to zurihac organizers and the Haskell community. I think having a bit of advertisement in advance would be nice. Your text reads like we are especially targeting people who have not yet found their way to the nixos discourse.

**Comment by cdepillabout** [comment]
@maralorn thanks, these are all good suggestions!

**Pull request closed without merging**